samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_eu.properties

```diff
@@ -355,3 +355,15 @@ answers_title=Erantzunak
 
 # S2U-9
 print_report=Inprimatu txostena
+
+# S2U-7
```

Comment:
@pablosanroman would you mind reviewing these strings please? Thank you very much

Reply:
Hi @mpellicer !

print_report=Inprimatu txostena
export_as_pdf=Esportatu PDFa
export_as_xlsx=Esportatu Excela
export_pdf=PDF gisa
export_xlsx=Excel gisa
export=Esportatu
skew_coef=Asimetria-koefizientea.
difficulty=Zailtasuna
total_attempts=Saiakerak, guztira
total_correct=Onak, guztira
total_incorrect=Txarrak, guztira

Reply:
Thank you @PabloSanRoman !

Reply:
@stetsche We have new translation strings